customizations/README.md

```diff
@@ -105,7 +105,7 @@ To add your own certificate you need:
   ```bash
   kubectl create secret tls \
     -n <namespace> \
-    <your_own_installation_name|carto>-tls-certificate \
+    mycarto-tls-certificate \
```

Comment:
And magically I remember something that @aitorch tell us.

This name is the same name generated inside the container so we should use another name to avoid collisions.

My suggestion: `mycarto-custom-tls-certificate` (That also applies to the other changes of this file)